src/ensembl/src/content/app/browser/drawer/Drawer.scss

```diff
@@ -14,30 +15,36 @@
     text-align: center;
   }
 
-  dd {
+  p {
+    margin: 0px;
+  }
+
+  .clearFix {
+    float: left;
```

Comment:
Please let's not lay this out with floats? Let's use something more robust and better fit for layout, such as CSS grid (see [example on jsfiddle](https://jsfiddle.net/o7y9dw2a/1/)) or flexbox (see [example on jsfiddle](https://jsfiddle.net/rq2vxn7e/)), or even a regular html table?

Reply:
Sure thing, Its just that float was already there so i just went with what was there to get it working.Because it would have been a bigger change that's why i didn't use the others. I didn't know it was that bad to use float; i did try moving the layout and it was fine.